src/agenda/index.tsx

```diff
@@ -401,13 +434,15 @@ export default class AgendaView extends Component {
     const openCalendarScrollPadPosition = !hideKnob && this.state.calendarScrollable && this.props.showClosingKnob ? agendaHeight + HEADER_HEIGHT : 0;
     const shouldAllowDragging = !hideKnob && !this.state.calendarScrollable;
     const scrollPadPosition = (shouldAllowDragging ? HEADER_HEIGHT : openCalendarScrollPadPosition) - KNOB_HEIGHT;
-    const scrollPadStyle = {
-      position: 'absolute',
-      width: 80,
-      height: KNOB_HEIGHT,
-      top: scrollPadPosition,
-      left: (this.viewWidth - 80) / 2
-    };
+    const scrollPadStyle = StyleSheet.create({
```

Comment:
Why create the stylesheet here and not in the style file? You can only add the dynamic style directly to the prop...